develop-docs/self-hosted/index.mdx

```diff
@@ -220,6 +222,6 @@ Keep in mind that all this setup uses single-nodes for all services, including K
 
 ## Licensing
 
-Sentry is using [FSL (Functional Source License)](https://fsl.software/). Although it's not an OSI-approved license, the source code of Sentry and its surrounding service & libraries for self-hosted deployment are open and [available on GitHub](https://github.com/orgs/getsentry/repositories?q=has%3Atag-production). For self-hosted, this license means that users can use Sentry and deploy it anywhere (even inside an enterprise ecosystem), but users are prohibited to sell deployed self-hosted Sentry as any kind of offering (SaaS or any kind of business model) and users are prohibited to be a direct competitor of Sentry by using Sentry's code that is FSL-licensed.  After two years, the license converts to Apache 2.0 or MIT.
+Sentry is [Fair Source](https://fair.io/) under the [FSL (Functional Source License)](https://fsl.software/). Note that Fair Source is not under the [OSI](https://opensource.org/) umbrella, it becomes literally open source under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0) after 2 years, and is practically open source before that except when you are a Sentry competitor. This license means that users can use Sentry and deploy it anywhere (even inside an enterprise ecosystem), but users are prohibited to sell deployed self-hosted Sentry as any kind of offering (SaaS or any other business model), and users are prohibited to be a direct competitor of Sentry by using Sentry's code that is FSL-licensed. The source code of Sentry and its surrounding service & libraries for self-hosted deployment are open and [available on GitHub](https://github.com/orgs/getsentry/repositories?q=has%3Atag-production).
```

Comment:
> The source code of Sentry and its surrounding service & libraries for self-hosted deployment are open and [available on GitHub](https://github.com/orgs/getsentry/repositories?q=has%3Atag-production).

Although I'd argue that not all repository that are tagged "production" is used by self-hosted, but eh, this is fine.

Reply:
We can probably tag all related repos with `self-hosted` or something? @hubertdeng123?